go/mysql/binlog_event.go

```diff
@@ -133,8 +136,9 @@ type BinlogEvent interface {
 	// IsPseudo is for custom implementations of GTID.
 	IsPseudo() bool
 
-	// IsCompressed returns true if a compressed event is found (binlog_transaction_compression=ON)
-	IsCompressed() bool
+	// IsTransactionPayload returns true if a compressed transaction
+	// payload event is found (binlog_transaction_compression=ON).
+	IsTransactionPayload() bool
```

Comment:
Curious, what's the reason behind this naming change? `IsTransactionPayload` on a cursory read doesn't seem to imply compression.

Reply:
Just because that's the type of event it is in MySQL.

Reply:
Makes sense.